### PR TITLE
Improve Readme

### PR DESCRIPTION
**Pull request by networkfusion**
## Motivation and Context
Options were hard to understand.

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [x] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Review by torbacz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -9,22 +9,21 @@
...
-`solutionsToCheck`||List of Solution(s) to update in the `workingDirectory` directory.
-`reposToUpdate`||List of repository(es) to update.
-`exclusionList`||List of solution(s) to exclude from update. Comma separated list.
-`gitHubUser`||Git hub user for creating PR.
-`gitHubEmail`||Git hub user email for creating PR.
-`gitHubAuth`||GitHub Personal Access Token for creating PR.
-`nugetConfig`||Path to nuget.config file to use for NuGet operations.
-`repoOwner`||Repository owner on git hub. github.com/**[repoOwner]**/repositoryName
-`useTokenForClone`||Should use token for cloning repositories. Required when working with private repositories.
+|Name               |Default                            |Options                        |Description
```
Comment (Ellerbach) [inline README.md:12]: Can you please form correctly the tbles? They should all start with `|` and all finishes with `|`. For the separation line, you need at least 3 same symbols, so 3 dahses like `---` . GitHub and VS Code are tolerent but it's far to be the case of all viewers.

Comment (networkfusion) [reply README.md]: Done.

**Review by Ellerbach — APPROVED** [review]
Looks good to me

**Comment by networkfusion** [comment]
Blocked until #161

**Pull request merged**